src/components/ui/IconButton/IconButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { IconButton as UIIconButton } from '@faststore/ui'
+import type { IconButtonProps as UIIconButtonProps } from '@faststore/ui'
+
+import './icon-button.scss'
+
+interface IconButtonProps {
+  classes?: string
+}
+
+type Props = IconButtonProps & UIIconButtonProps
+
+function IconButton({ icon, classes, ...props }: Props) {
+  return (
+    <UIIconButton className={`icon-button ${classes}`} icon={icon} {...props} />
```

Comment:
We normally call the rest params this way.

```suggestion
function IconButton({ icon, classes, ...otherProps }: Props) {
  return (
    <UIIconButton className={`icon-button ${classes}`} icon={icon} {...otherProps} />
```